### DISCUSSION TITLE
Any toggle event for Inertial Separator?

### DISCUSSION THREAD

**Discussion by beckerdo** [nguyenquyhy/Flight-Tracker-StreamDeck#227] [General]
I've searched for "inertial" and "separator" and variants, but I cannot find an event for inertial separator.

This is a toggle switch that some turbine planes such as the Daher TBM 930 have to prevent dirt and gravel from being ingested into the turbine.

It is one of the few switches on the TBM that I cannot flip with with this software. Thanks, Dan

**Comment by Foreigner2000** [comment]
I use this value "ANTI_ICE_TOGGLE_ENG1" for the inert sep on the TBM 930.
The actual switch in the cockpit doesnt respond with this command but the MFD reacts on it.
